.github/workflows/tests_linux.yml

```diff
@@ -80,7 +80,8 @@ jobs:
         run: |
           cd main/
           pytest tests/
-          pl-device-test --device lightning.kokkos
+          pl-device-test --device lightning.kokkos --skip-ops --shots=20000
+          pl-device-test --device lightning.kokkos --shots=None --skip-ops
```

Comment:
Is there a reason for the reverse order in the arguments here?

Reply:
Not really, just followed previous implementation.